server/src/main/java/org/apache/uniffle/server/ShuffleServerConf.java

```diff
@@ -263,6 +264,15 @@ public class ShuffleServerConf extends RssBaseConf {
       .defaultValue(64L * 1024L * 1024L)
       .withDescription("For multistorage, the event size exceed this value, flush data  to cold storage");
 
+  public static final ConfigOption<StorageManagerSelector.ColdStoragePreferredFactor>
+      MULTISTORAGE_SELECTOR_COLD_STORAGE_PREFER = ConfigOptions
+      .key("rss.server.multistorage.cold.storage.preferred.factor")
```

Comment:
Could you give this config option a better name? It is more like a strategy in our system.

Reply:
WDYT?

Reply:
rss.server.multistorage.selector.strategy

Reply:
This has been discussed in above conversation. https://github.com/apache/incubator-uniffle/pull/621#discussion_r1111945895. If you prefer, I will refactor it.

Reply:
It's better to refactor ... it's more clear.